compressed_image_transport/src/compressed_publisher.cpp

```diff
@@ -51,29 +53,37 @@ namespace enc = sensor_msgs::image_encodings;
 namespace compressed_image_transport
 {
 
-void CompressedPublisher::advertiseImpl(ros::NodeHandle &nh, const std::string &base_topic, uint32_t queue_size,
-                                        const image_transport::SubscriberStatusCallback &user_connect_cb,
```

Comment:
We are not porting the SubscriberStatusCallback to ROS2. Looking for the similiar feature I found a [answer.ros.org](https://answers.ros.org/question/286069/implementing-subscriberstatuscallback-within-ros2/) by William explaining in March that the feature is not implemented yet. I assume that it is still unimplemented.